server/grpc_service.go

```diff
@@ -1709,8 +1709,19 @@ func (s *GrpcServer) ScatterRegion(ctx context.Context, request *pdpb.ScatterReg
 	if err != nil {
 		return nil, err
 	}
-	if op != nil {
-		rc.GetOperatorController().AddOperator(op)
+
+	if op == nil {
+		return &pdpb.ScatterRegionResponse{
+			Header: s.wrapErrorToHeader(pdpb.ErrorType_UNKNOWN,
```

Comment:
It should not be an error. for example, if there are 6 tikv nodes， scatter a batch of regionscontinuously, 50% of the ops should be nil.

Reply:
it come here for the scatter of one region only.